src/main/scala/com/microsoft/sqlserver/jdbc/spark/failureinjection/FailureInjection.scala

```diff
@@ -1,22 +1,20 @@
 /*
- * Copyright (c) 2018 Microsoft Corporation
+ * Copyright (c) 2020 Microsoft Corporation
```

Comment:
@rajmera3 usually most ASL based projects do not include a copyright line, it is usually just the boilerplate header. Any comments?

Reply:
@rajmera3 - can you clarify my question above?

Reply:
@arvindshmicrosoft You're right - we should remove the MS copyright headers

Reply:
@cchighman - sorry, could you please remove the copyright headers as per this comment thread? Thanks again for your follow-up.

Reply:
Yes